### PR TITLE
Removes irrelevant quirk restriction of a nonexistant quirk

### PR DESCRIPTION
**Pull request by nikothedude**
## About The Pull Request

Title.

## How This Contributes To The Skyrat Roleplay Experience

Chunky fingers doesnt exist anymore.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
it compiles
</details>

## Changelog

:cl:
del: Removes a irrelevant quirk restriction of a nonexistant quirk
/:cl:

### CODE REVIEW

File: code/__DEFINES/~skyrat_defines/jobs.dm [hunk 1]
```diff
@@ -6,9 +6,9 @@
 #define JOB_UNAVAILABLE_FLAVOUR (JOB_UNAVAILABLE_LANGUAGE + 1)
 #define JOB_UNAVAILABLE_AUGMENT (JOB_UNAVAILABLE_FLAVOUR + 1)
 
-#define SEC_RESTRICTED_QUIRKS "Blind" = TRUE, "Brain Tumor" = TRUE, "Deaf" = TRUE, "Paraplegic" = TRUE, "Hemiplegic" = TRUE, "Mute" = TRUE, "Foreigner" = TRUE, "Pacifist" = TRUE, "Chunky Fingers" = TRUE, "No Guns" = TRUE, "Illiterate" = TRUE, "Nerve Stapled" = TRUE
-#define HEAD_RESTRICTED_QUIRKS "Blind" = TRUE, "Deaf" = TRUE, "Mute" = TRUE, "Foreigner" = TRUE, "Chunky Fingers" = TRUE, "Brain Tumor" = TRUE, "Illiterate" = TRUE
-#define TECH_RESTRICTED_QUIRKS "Chunky Fingers" = TRUE
+#define SEC_RESTRICTED_QUIRKS "Blind" = TRUE, "Brain Tumor" = TRUE, "Deaf" = TRUE, "Paraplegic" = TRUE, "Hemiplegic" = TRUE, "Mute" = TRUE, "Foreigner" = TRUE, "Pacifist" = TRUE, "No Guns" = TRUE, "Illiterate" = TRUE, "Nerve Stapled" = TRUE
+#define HEAD_RESTRICTED_QUIRKS "Blind" = TRUE, "Deaf" = TRUE, "Mute" = TRUE, "Foreigner" = TRUE, "Brain Tumor" = TRUE, "Illiterate" = TRUE
+#define TECH_RESTRICTED_QUIRKS
```
Comment (vinylspiders) [inline code/__DEFINES/~skyrat_defines/jobs.dm]:
```suggestion
```
This is nothing now, just get rid of it (as well as any references to it).

**Review by vinylspiders — CHANGES_REQUESTED** [review]
Nice catch, could you also remove the other lingering references to this quirk while you're at it?

`var/allow_chunky` and `var/chunky_finger_usable`

**Comment by nikothedude** [comment]
> `allow_chunky`

No, the trait still exists and is obtainable via ashies and shit, therfore its relevant

**Comment by vinylspiders** [comment]
> > `allow_chunky`
> 
> No, the trait still exists and is obtainable via ashies and shit, therfore its relevant

That is confusing. I see.

**Review by vinylspiders — APPROVED** [review]

**Pull request merged**